netbox/dcim/filtersets.py

```diff
@@ -1583,13 +1646,23 @@ class InterfaceFilterSet(
         to_field_name='name',
         label='Virtual Device Context',
     )
+    wireless_lan_id = django_filters.ModelMultipleChoiceFilter(
+        field_name='wireless_lans',
+        queryset=WirelessLAN.objects.all(),
+        label='Wireless LAN',
```

Comment:
internationalize

Reply:
Found a few instances where we neglected to wrap labels with `gettext()`; going to clean them all up.